<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>990</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QPushButton" name="PBmakeObject">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>940</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Object</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBplatform1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Platform</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/pavement1.png</normaloff>:/images/pavement1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBplatform2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Platform</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/pavement2.png</normaloff>:/images/pavement2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBplatform3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Platform</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/pavement3.png</normaloff>:/images/pavement3.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBplatform4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Platform</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/pavement4.png</normaloff>:/images/pavement4.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBplatform6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Platform</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/pavement6.png</normaloff>:/images/pavement6.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBplatform7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Platform</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/pavement7.png</normaloff>:/images/pavement7.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBplatform5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Platform</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/pavement5.png</normaloff>:/images/pavement5.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBplatform8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Platform</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/pavement8.png</normaloff>:/images/pavement8.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBplatform10">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Platform</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/pavement10.png</normaloff>:/images/pavement10.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBplatform11">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Platform</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/pavement11.png</normaloff>:/images/pavement11.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBplatform9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Platform</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/pavement9.png</normaloff>:/images/pavement9.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBplatform12">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Platform</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/pavement12.png</normaloff>:/images/pavement12.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBplatform14">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Platform</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/pavement14.png</normaloff>:/images/pavement14.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBplatform15">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Platform</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/pavement15.png</normaloff>:/images/pavement15.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBplatform13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Platform</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/pavement13.png</normaloff>:/images/pavement13.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBplatform16">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Platform</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/pavement16.png</normaloff>:/images/pavement16.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBplayer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Player</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/maincharacter/stand.png</normaloff>:/images/maincharacter/stand.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBcoin">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Coin</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/goldCoin/goldCoin5.png</normaloff>:/images/goldCoin/goldCoin5.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Player</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>85</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Platform</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Coin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>85</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBitem">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Platform</string>
   </property>
   <property name="text">
    <string>Item</string>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LEitem">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>530</y>
     <width>201</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>85</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Enemy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBenemyGround">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>620</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Platform</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/groundrobot.png</normaloff>:/images/groundrobot.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBenemyFly">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>620</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Platform</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/flyingrobot.png</normaloff>:/images/flyingrobot.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBwin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Win</string>
   </property>
   <property name="text">
    <string>Win</string>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Win</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>780</y>
     <width>85</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>xSpeed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>820</y>
     <width>85</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ySpeed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>860</y>
     <width>85</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>xRange</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>900</y>
     <width>85</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>yRange</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>700</y>
     <width>85</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LExSpeed">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>780</y>
     <width>113</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LEySpeed">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>820</y>
     <width>113</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LExRange">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>860</y>
     <width>113</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LEyRange">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>900</y>
     <width>113</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LEwidth">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>700</y>
     <width>113</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="QWworld" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(168, 166, 166);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>740</y>
     <width>85</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LEheight">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>740</y>
     <width>113</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PBdelete">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>940</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Resize World</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>16</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>16</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LEresizeX">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>113</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LEresizeY">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>113</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PBresize">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>120</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Resize</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
